pkgs/development/python-modules/airlift/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, poetry-core, argparse, halo, pyyaml, hiyapyco, jinja2, dotmap, requests
+, kubernetes-helm, kind, docker, pythonOlder, buildPythonPackage
+, fetchFromGitHub }:
+
+buildPythonPackage rec {
+
+  disabled = pythonOlder "3.9";
+  pname = "airlift";
+  format = "pyproject";
```

```suggestion
  pyproject = true;
```
Please use `pyproject = true` instead.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters